packages/perseus/src/util/extract-perseus-data.ts

```diff
@@ -59,9 +59,9 @@ function getAnswersFromWidgets(
                     categorizer.options?.items &&
                     categorizer.options?.values
                 ) {
-                    const categories = categorizer.options?.categories;
-                    const items = categorizer.options?.items;
-                    const values = categorizer.options?.values;
+                    const categories = categorizer.options.categories;
+                    const items = categorizer.options.items;
+                    const values = categorizer.options.values;
```

Comment:
These `?.` nullish coalescing operators were not needed, since we check the nullishness of each of these fields above.